lib/pages/class_feedback/service/remote_config.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+
+const String _feedbackEnabled = 'feedback_enabled';
+
+class RemoteConfigService {
+  RemoteConfigService({RemoteConfig remoteConfig})
+      : _remoteConfig = remoteConfig;
+  final RemoteConfig _remoteConfig;
+  final defaults = <String, dynamic>{_feedbackEnabled: false};
+  static RemoteConfigService _instance;
+
+  static Future<RemoteConfigService> getInstance() async {
+    return _instance ??= RemoteConfigService(
+      remoteConfig: await RemoteConfig.instance,
+    );
+  }
+
+  bool get feedbackEnabled =>
+      // ignore: avoid_bool_literals_in_conditional_expressions
+      _remoteConfig == null ? true : _remoteConfig.getBool(_feedbackEnabled);
```

Comment:
Don't ignore warnings. And shouldn't this be "false" by default? This could be something like `_remoteConfig?.getBool(_feedbackEnabled) ?? false`. Or even better, since you actually have a `defaults` map, `_remoteConfig?.getBool(_feedbackEnabled) ?? defaults[_feedbackEnabled]`.